packages/scm/src/browser/scm-widget.tsx

```diff
@@ -115,15 +114,30 @@ export class ScmWidget extends ScmNavigableListWidget<ScmResource> implements St
         this.scmService.onDidChangeSelectedRepositories(repository => {
             if (repository) {
                 this.selectedRepoUri = repository.provider.rootUri;
-                this.title.label = 'Source Control: ' + repository.provider.contextValue;
-                this.shell.leftPanelHandler.refresh();
+                this.title.label = ScmWidget.LABEL + ': ' + repository.provider.contextValue;
+                const area = this.shell.getAreaFor(this);
```

Comment:
BTW why do we need to refresh an area? I've tried to remove it and could not break anything

Reply:
It is used to update the title of the tab